py-geopolars/python/geopolars/internals/geodataframe.py

```diff
@@ -1,23 +1,53 @@
-import polars
+from __future__ import annotations
 
+import polars as pl
+
+from geopolars import geopolars as core  # type: ignore
 from geopolars.internals.geoseries import GeoSeries
 
 DEFAULT_GEO_COLUMN_NAME = "geometry"
 
 
-class GeoDataFrame(polars.DataFrame):
+class GeoDataFrame(pl.DataFrame):
```

Comment:
Sure. Tell me if I should change it. I just assumed that because for pandas we use `pd` in `example.py` is used `import geopolars as gpl`. If there are some set conventions i would gladly use it.